ProcessMaker/Console/Commands/Install.php

```diff
@@ -153,36 +141,23 @@ public function handle()
         Storage::disk('install')->put('.env', $contents);
 
         // Install migrations
-        $this->call('migrate:fresh', [
+        $this->callSilent('migrate:fresh', [
             '--seed' => true,
         ]);
-        
-        // Seed default user groups if desired
-        if ($seedDefaultGroups) {
-            $this->call('db:seed', [
-                '--class' => 'DefaultGroupSeeder',
-            ]);        
-        }
+
+        $this->info(__("ProcessMaker database installed successfully."));
 
         // Generate passport secure keys and personal token oauth client
         $this->call('passport:install', [
             '--force' => true
         ]);
-		
+
 		//Create a symbolic link from "public/storage" to "storage/app/public"
         $this->call('storage:link');
 
-        // Install cron job for Laravel scheduled tasks
-        $this->info(__('Installing scheduled tasks...'));
-        $this->installCronJob();
-        
-        // Install Docker executors
-        $this->info(__('Installing Docker executors...'));        
-        $this->installDockerImages();
-        
         // Restart services so they pick up the new settings
-        $this->info(__('Restarting Services...'));
-        $this->info(
+        $this->info(__('Restarting services...'));
+        $this->info(e
```

Comment:
Extra character "e" is causing some issues here
```suggestion
        $this->info(
```

Reply:
Other than that change above this looks good and functions as intended.

Reply:
No services should be handled by the installer.  What's it trying to restart?

Reply:
I've removed it